drivers/volume/linstor/linstor.go

```diff
@@ -251,7 +251,12 @@ func (l *linstor) GetNodes() ([]*storkvolume.NodeInfo, error) {
 	return infos, nil
 }
 
-func (l *linstor) GetPodVolumes(podSpec *v1.PodSpec, namespace string) ([]*storkvolume.Info, error) {
+func (l *linstor) GetPodVolumes(podSpec *v1.PodSpec, namespace string, includePendingWFFC bool) ([]*storkvolume.Info, []*storkvolume.Info, error) {
+	// TODO how to handle includePendingWFFC
+	if includePendingWFFC {
```

Comment:
@adityadani What's the right way to handle for linstor?

Reply:
I would suggest not handing the boolean here and keep the behavior as it was before.
Can someone from the Linstor team review this patch and check if you need to handle this for LinStor driver?  cc @kvaps @chrboe

Reply:
So remove the `if` and completely ignore it. Fine with me

Reply:
@kvaps which old version? The version prior to these changes should have the same problem with pending WaitForFirstConsumer pvc's.

Reply:
we use [`github.com/libopenstorage/stork v1.4.1-0.20220119164932-78de9f1d48ab`](https://github.com/piraeusdatastore/linstor-scheduler-extender/blob/0a67ed34c935a5e846e4b232282c43474381e68c/go.mod#L53). I heard about this problem already, but I never faced with this issue personally.
Dunno why.

I think I think this will be enough:

```diff
--- a/pkg/driver/linstor.go
+++ b/pkg/driver/linstor.go
@@ -268,7 +268,7 @@ func (l *linstor) GetPodVolumes(podSpec *v1.PodSpec, namespace string, includePe
                                continue
                        }

-                       if pvc.Status.Phase == v1.ClaimPending {
+                       if !includePendingWFFC && pvc.Status.Phase == v1.ClaimPending {
                                return nil, nil, &storkvolume.ErrPVCPending{
                                        Name: volume.PersistentVolumeClaim.ClaimName,
                                }
```

however I need a time to confirm this

Reply:
@lpitstickpstg please take a look
https://github.com/libopenstorage/stork/pull/1100

changes are already tested

Reply:
Thanks @kvaps !